include/highfive/bits/H5Converter_misc.hpp

```diff
@@ -46,7 +46,7 @@ inline bool is_1D(const std::vector<size_t>& dims)
 
 inline size_t compute_total_size(const std::vector<size_t>& dims)
 {
-    return std::accumulate(dims.begin(), dims.end(), static_cast<size_t>(1),
```

Comment:
It looks like the cast here is needed, otherwise we still get a warning when compiling on win32, where `size_t` > `unsigned int`. I'll send a follow-up PR.

Reply:
Send #179 to fix it, please have a look.